packages/app/src/files-manager/helpers/check-files-extensions.helper.ts

```diff
@@ -5,10 +5,19 @@
 
 import { eligibleFormat } from "App/files-manager/constants/eligible-format.constant"
 
-export const checkFilesExtensions = (filesPaths: string[]): boolean => {
-  return filesPaths.every((filePath) => {
+export const checkFilesExtensions = (
+  filesPaths: string[]
+): [string[], string[]] => {
+  const validFiles = filesPaths.filter((filePath) => {
     return eligibleFormat.includes(
       (filePath.split(".").pop() ?? "").toLocaleLowerCase()
     )
   })
+  const invalidFiles = filesPaths.filter((filePath) => {
+    return !eligibleFormat.includes(
+      (filePath.split(".").pop() ?? "").toLocaleLowerCase()
+    )
+  })
+
+  return [validFiles, invalidFiles]
```

Comment:
Maybe it would ease things to create function:
const isPathEligible = (path: string) => eligibleFormat.includes(
      (path.split(".").pop() ?? "").toLocaleLowerCase()
      
 and then use it in both filters? :D

Reply:
While writing this i had in mind the syntax of useState for example where as a return value you get something similar to tuple.

Reply:
still I think that object might be more descriptive + the filter function is a great idea. Working on it.